service/s3/api_op_WriteGetObjectResponse.go

```diff
@@ -369,6 +375,85 @@ func (c *Client) addOperationWriteGetObjectResponseMiddlewares(stack *middleware
 	return nil
 }
 
+type opWriteGetObjectResponseResolveEndpointMiddleware struct {
```

Comment:
NOTE: I'm going to comment on this generated middleware for no reason in particular, consider it a review of all generated middleware though.